database/seeds/eggs/minecraft/egg-paper.json

```diff
@@ -3,12 +3,12 @@
     "meta": {
         "version": "PTDL_v1"
     },
-    "exported_at": "2020-11-03T04:22:56+00:00",
+    "exported_at": "2020-12-04T12:08:20-05:00",
     "name": "Paper",
     "author": "parker@pterodactyl.io",
     "description": "High performance Spigot fork that aims to fix gameplay and mechanics inconsistencies.",
-    "features": ["eula"],
-    "image": "quay.io\/pterodactyl\/core:java",
+    "features": null,
+    "image": "quay.io\/parkervcp\/pterodactyl-images:debian_openjdk-11",
```

Comment:
Unrelated, can we merge these images into the core repo?

Reply:
I can pr a java-11 image up to the core images repo